src/isc-dhcp/.gitignore

```diff
@@ -0,0 +1,4 @@
+*
+!.gitignore
+!Makefile
+!patch/
```

Comment:
Who ignored it?

Reply:
The first line, `*` ignores everything, then we un-ignore the Makefile here.